_codux/boards/components/product-card/product-card.board.tsx

```diff
@@ -1,15 +1,34 @@
 import { createBoard } from '@wixc3/react-board';
 import { ProductCard } from '~/components/product-card/product-card';
+import styles from './product-card.board.module.scss';
 
 export default createBoard({
     name: 'Product Card',
     Board: () => (
-        <ProductCard
-            name='Shel 50" Class LED 4K UHD Smart TV'
-            price={{ formatted: { price: '$85' } }}
-            style={{ width: '300px' }}
-            imageUrl="https://wixmp-b7f7090100b13623109851bc.wixmp.com/layouters-starters/img_02.jpg"
-        />
+        <div className={styles.root}>
+            <ProductCard
```

Comment:
Do you have `ProductCard` in UI-Kit board?

Reply:
you're right, removed separate board and added 'Out of stock' and 'With discount' cards to the UI kit board.
Even though personally I don't like the idea :)